### PR TITLE
doc improvement: use acronym GNSS instead of GPS where appropriate

### PR DESCRIPTION
**Pull request by tokangas**
nRF9160 satellite navigation supports also QZSS satellites, so the acronym GNSS should used instead of GPS when not talking explicitly about the GPS system.

### CODE REVIEW

File: doc/nrf/libraries/networking/nrf_cloud_pgps.rst [hunk 1]
```diff
@@ -13,20 +13,20 @@ It can be used with or without Assisted GPS (`A-GPS`_) data from nRF Cloud.
...
 
 .. note::
 
    If two-week prediction sets are used, TTFF towards the end of the second week will increase due to accumulated errors in the predictions and decrease in number of satellite ephemerides in the later prediction periods.
 
-If nRF Cloud services such as A-GPS or P-GPS are used either individually or in combination, the broadcasted information and predictions of satellite data can be downloaded at a faster rate from nRF Cloud than from the GPS satellites.
+If nRF Cloud services such as A-GPS or P-GPS are used either individually or in combination, the broadcasted information and predictions of satellite data can be downloaded at a faster rate from nRF Cloud than from the satellites.
 
-The use of P-GPS reduces Time to First Fix (TTFF) (time for a GPS device to estimate its position) when compared to using no assistance at all.
+The use of P-GPS reduces Time to First Fix (TTFF) (time for a GNSS device to estimate its position) when compared to using no assistance at all.
```
Comment (peknis) [inline doc/nrf/libraries/networking/nrf_cloud_pgps.rst:29]: GNSS device or module?

Comment (tokangas) [reply doc/nrf/libraries/networking/nrf_cloud_pgps.rst:29]: I changed device to module in some places, but I missed that one. I'll update it.

**Review by peknis — APPROVED** [review]

**Review by coderbyheart — APPROVED** [review]
**nitpick:** A changelog entry would be nice, so users can discover the new term when searching for GPS.

**Review by VeijoPesonen — APPROVED** [review]
Looks good to me when it comes to LwM2M client documentation.

**Review by rlubos — APPROVED** [review]

**Comment by tokangas** [comment]
Added a changelog entry. @peknis, please have a look.

**Review by simensrostad — APPROVED** [review]

File: applications/asset_tracker/README.rst [hunk 2]
```diff
@@ -158,13 +158,13 @@ On the Thingy:91, the application state is indicated by a single RGB LED as foll
    * - Blue
      - Connected, sending environment data
    * - Purple
-     - Searching for GPS
+     - Searching for GNSS
```
Comment (hiltunent) [inline applications/asset_tracker/README.rst:161]:
```suggestion
     - Acquiring GNSS position 
```

File: doc/nrf/libraries/networking/nrf_cloud_agps.rst [hunk 3]
```diff
@@ -50,13 +50,13 @@ Since the library requests only partial data, data traffic reduction and battery
 The duration for which a particular type of assistance data is valid is different for each type of assistance data.
 As an example, `Almanac`_ data has a far longer validity (several months) than `Ephemeris`_ data (2 to 4 hours).
 
-Since the library receives a partial assistance data set, it may cause GPS to download the missing data from satellites.
+Since the library receives a partial assistance data set, it may cause GNSS to download the missing data from satellites.
 
 When A-GPS data is downloaded using LTE network, the LTE link is in `RRC connected mode <RRC idle mode_>`_.
-The GPS can only operate only when the device is in RRC idle mode.
+The GNSS module can only operate only when the device is in RRC idle mode.
```
Comment (hiltunent) [inline doc/nrf/libraries/networking/nrf_cloud_agps.rst:56]:
```suggestion
The GNSS module can only operate when the device is in RRC idle mode.
```

File: doc/nrf/ug_thingy91.rst [hunk 4]
```diff
@@ -59,11 +59,11 @@ Operating modes
...
 Thingy:91 contains RGB indicator LEDs, which indicate the operating state of the device as described in :ref:`operating states of Thingy:91<thingy91_operating_states>`.
 
-GPS
-===
+GNSS
+====
 
-Thingy:91 has GPS, which, if activated, allows the device to be located globally using GPS signals.
-To activate GPS, long-press the SW3 button.
+Thingy:91 has GNSS, which, if activated, allows the device to be located globally using GNSS signals.
```
Comment (hiltunent) [inline doc/nrf/ug_thingy91.rst:65]:
```suggestion
Thingy:91 has a GNSS receiver, which, if activated, allows the device to be located globally using GNSS signals.
```

**Review by hiltunent — CHANGES_REQUESTED** [review]

**Review by junqingzou — APPROVED** [review]
Looks good with SLM doc

File: doc/nrf/libraries/networking/nrf_cloud_agps.rst (hunk 3 above)
Comment (martiles) [reply doc/nrf/libraries/networking/nrf_cloud_agps.rst:56]: and in Power Saving Mode

File: applications/asset_tracker/README.rst (hunk 2 above)
Comment (tokangas) [reply applications/asset_tracker/README.rst]: Done.

File: doc/nrf/libraries/networking/nrf_cloud_agps.rst (hunk 3 above)
Comment (tokangas) [reply doc/nrf/libraries/networking/nrf_cloud_agps.rst]: Done. Added also PSM.

File: doc/nrf/ug_thingy91.rst (hunk 4 above)
Comment (tokangas) [reply doc/nrf/ug_thingy91.rst]: Done.

**Review by hiltunent — APPROVED** [review]

**Review by lemrey — APPROVED** [review]

**Review by jtguggedal — APPROVED** [review]

**Pull request merged**